src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1211,10 +1211,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return static_cast<uint16_t>(_actualFont.GetWeight());
     }
 
-    til::size ControlCore::FontSizeInDips() const
+    winrt::Windows::Foundation::Size ControlCore::FontSizeInDips() const

```

Comment:
Are we not using `til::size` because it does not use floats?

Reply:
Yep! I was a bit confused about this bit, as FontSizeInDips is usually not an integer on any display scale but 100%.

Reply:
The extra fun bit is that `ControlCore::FontSize()` without the `InDips` already returned a `winrt::Windows::Foundation::Size`, just not this function. 😅